src/cluster.c

```diff
@@ -4207,6 +4214,54 @@ sds clusterGenNodeDescription(clusterNode *node) {
     return ci;
 }
 
+/* Generate all nodes slots topology info. Generating one node slots info is
+ * costly in clusterGenNodeDescription(), if there are many nodes in cluster,
+ * it will cost much time and block server to separately generate every node
+ * slots info. We have all slots topology of cluster, so we can generate all
+ * nodes slots info by only iterating slots topology once. */
```

Comment:
"Generate the slot topology for all nodes and store the string representation in the slots_info struct on the node. This is used to improve the efficiency of clusterGenNodeDescription() because it removes looping of the slot space for generating the slot info for each node individually." ?

Reply:
@madolson Thanks, clearer than my description